browser/src/GenePage/GenePage.tsx

```diff
@@ -117,6 +126,35 @@ const LegendSwatch = styled.span`
   }
 `
 
+type TableName = 'constraint' | 'cooccurrence'
+
+type TableSelectorProps = {
+  ownTableName: TableName
+  selectedTableName: TableName
+  setSelectedTableName: Dispatch<SetStateAction<TableName>>
+}
+
+const BaseTableSelector = styled.div<TableSelectorProps>
+const TableSelector = BaseTableSelector.attrs(
+  ({ setSelectedTableName, ownTableName }: TableSelectorProps) => ({
+    onClick: () => setSelectedTableName(ownTableName),
+  })
+)`
+  border: 1px solid black;
+  border-radius: 0.5em;
+  cursor: pointer;
+  font-weight: bold;
```

Comment:
It looks like `SegmentedControl` can't in fact contain an `InfoButton` (the component that corresponds to those question-mark icons), and it would need significant changes to make it able to. Given that it makes more sense to me to just match the font weight itself for now and revisit `SegmentedControl` later.

Reply:
Also the gray background color was slightly off from the dataset button, changed that as well.